src/sgame/sg_cm_world.cpp

```diff
@@ -113,14 +113,8 @@ clipHandle_t G_CM_ClipHandleForEntity( const gentity_t *ent )
 		return CM_InlineModel( ent->s.modelindex );
 	}
 
-	if ( ent->r.svFlags & SVF_CAPSULE )
-	{
-		// create a temp capsule from bounding box sizes
-		return CM_TempBoxModel( ent->r.mins, ent->r.maxs, true );
-	}
-
 	// create a temp tree from bounding box sizes
```

Comment:
I don't understand this comment but maybe it's better to get rid of it since it was on only one of the cases?

Reply:
I've updated the comment to say "tree/capsule"